glom/test/test_trace.py

```diff
@@ -0,0 +1,54 @@
+import traceback
+
+import pytest
+
+from glom import glom, S, Coalesce, GlomError
+from glom.trace import line_stack, short_stack, tall_stack
+
+
+from glom.tutorial import CONTACTS
+
+
+def test_line_trace():
+    stacklifier = ([{'data': S}],)
+    scope = glom([1], stacklifier)[0]['data']
+    line_stack(scope)
```

Comment:
are we going to want to test these against some values?

Reply:
I'm not sure what the invariant is other than just grab the current value

Reply:
I added in what I meant, just that line_stack(scope) == "the expected string"